Sources/Beacon.Core/Beacon.Core.csproj

```diff
@@ -25,7 +25,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
-    <LangVersion>7</LangVersion>
+    <LangVersion>6</LangVersion>
```

Comment:
Why did you downgrade the language version?

Reply:
The PSake build did not work with 7. Did not investigate it a lot deeper, took the easy way and downgraded it.

Reply:
Ah, ok. I remember that problem. I'll fix that later.